src/main/java/com/zendesk/maxwell/schema/MysqlSavedSchema.java

```diff
@@ -675,43 +676,42 @@ private void fixUnsignedColumns(Schema recaptured) throws SQLException, InvalidS
 		}
 	}
 
-	private void fixColumnCases(Schema recaptured) throws SQLException {
+	private void fixColumnCases(Schema recaptured) throws InvalidSchemaError {
```

Comment:
I'm going to start a review, but this is a big PR and we'll have to go in chunks.